### DISCUSSION TITLE
Unstyled HeadlessUI Figma Design System / Component Library starter

### DISCUSSION THREAD

**Discussion by connorwforsyth** [tailwindlabs/headlessui#2234] [Ideas]
As a designer / dev who uses headless ui and tailwind ui it would be awesome to have a Figma component library to build out prototype ui's faster!

Not sure if this is in the works. But I think it would be a great companion to tailwind resources. ✌️

**Comment by SeanDunford** [comment]
There is an unofficial Figma plugin it would be nice to see this or something similar adopted as the "official" design file though. 

https://www.figma.com/community/file/1238498101687808134/Tailwind-Headless-UI-with-Animations
